docs/source/frameworks/catboost.rst

```diff
@@ -0,0 +1,169 @@
+   )
+
+   # train the model
+   model.fit(X, y)
+
+
+After training, use :obj:`~bentoml.catboost.save_model` to save the model instance to BentoML model store.
+
+.. code-block:: python
+
+   bento_model = bentoml.catboost.save_model("catboost_cancer_clf", model)
+
+
+To verify that the saved learner can be loaded properly:
+
+.. code-block:: python
+
+   model = bentoml.catboost.load_model("catboost_cancer_clf:latest")
+
+   model.predict(cbt.Pool([[1.308e+01, 1.571e+01, 8.563e+01, 5.200e+02, 1.075e-01, 1.270e-01,
+       4.568e-02, 3.110e-02, 1.967e-01, 6.811e-02, 1.852e-01, 7.477e-01,
+       1.383e+00, 1.467e+01, 4.097e-03, 1.898e-02, 1.698e-02, 6.490e-03,
+       1.678e-02, 2.425e-03, 1.450e+01, 2.049e+01, 9.609e+01, 6.305e+02,
+       1.312e-01, 2.776e-01, 1.890e-01, 7.283e-02, 3.184e-01, 8.183e-02]]))
+
+
+Building a Service using CatBoost
+--------------------------------
+
+.. seealso::
+
+   :ref:`Building a Service <concepts/service:Service and APIs>`: more information on creating a prediction service with BentoML.
+
+.. code-block:: python
+
+   import bentoml
+
+   import numpy as np
+
+   from bentoml.io import NumpyNdarray
+
+   runner = bentoml.catboost.get("catboost_cancer_clf:latest").to_runner()
+
+   svc = bentoml.Service("cancer_clf", runners=[runner])
+
+
+   @svc.api(input=NumpyNdarray(), output=NumpyNdarray())
+   def classify_cancer(input: np.ndarray) -> np.ndarray:
+      # returns sentiment score of a given text
+      res = runner.predict.run(input)
+      return res
+
+
+When constructing a :ref:`bentofile.yaml <concepts/bento:Bento Build Options>`,
+there are two ways to include CatBoost as a dependency, via ``python`` or
+``conda``:
+
+.. tab-set::
+
+   .. tab-item:: python
+
+      .. code-block:: yaml
+
+         python:
+         - catboost
+
+   .. tab-item:: conda
+
+      .. code-block:: yaml
+
+         conda:
+           channels:
+           - conda-forge
+           dependencies:
+           - catboost
+
+
+Using Runners
+-------------
+
+.. seealso::
+
+   See :ref:`concepts/runner:Using Runners` doc for a general introduction to the Runner concept and its usage.
+
+A runner for a Booster is created like so:
+
+.. code-block:: python
+
+   bentoml.catboost.get("model_name:model_version").to_runner()
+
+``runner.predict.run`` is generally a drop-in replacement for ``model.predict``. However, while it
+is possible to pass a ``Pool`` as input, BentoML does not support adaptive batching in that case.
+It is therefore recommended to use a NumPy ``ndarray`` or Pandas ``DataFrame`` as input instead.
+
+.. note
+
+   BentoML only support ``predict`` and ``predict_proba`` as a signature for now. ``staged_predict`` is not yet supported.
+
+Using GPU
+---------
+
+If there is a GPU available, the CatBoost Runner will automatically use ``task_type=GPU`` by default.
+This can be disabled by using the
+:ref:`BentoML configuration file <guides/configuration:Configuring BentoML>` to disable Runner GPU
+access:
```

Comment:
```suggestion
CatBoost Runner will automatically use ``task_type=GPU`` if GPU is available on the system.
This behaviour can then be disabled via :ref:`BentoML's configuration<guides/configuration:Configuring BentoML>`:
access:
```
```suggestion
If there is a GPU available, the CatBoost Runner will automatically use ``task_type=GPU`` by default.
This can be disabled by using the
:ref:`BentoML configuration file <guides/configuration:Configuring BentoML>` to disable Runner GPU
access:
```

Reply:
I think the old wording was fine?